explore/src/main/scala/explore/EditableLabel.scala

```diff
@@ -139,13 +142,15 @@ object EditableLabel {
         if (editing.value.value)
           <.div(^.width := "100%", ^.display.flex)(
             InputText(
-              id = "editable-label-input", // won't necesessarily be unique...
+              id = s"editable-label-input-${props.id}",
```

Comment:
Has the new hooks in scalajs-react been released yet? If so you could also use the `useId` for a unique id per component, rather than having to pass id's as a prop. This situation is exactly what it was made for

Reply:
Yes! It should be in beta 3. I had forgotten about that.